app/components/balance-donut/template.hbs

```diff
@@ -1,40 +1,42 @@
-<div class="donut-title">{{this.title}}</div>
+<div style={{this.style}} class="user-header-absence-balance balance-donut {{this.color}}">
```

Comment:
```suggestion
<div {{style this.style}} class="user-header-absence-balance balance-donut {{this.color}}">
```

Let's use the [ember-style-modifier](https://github.com/jelhan/ember-style-modifier) since we already have it.